examples/eco/07-integrated-hydrogen/input/floris/floris_input_iea_18MW_osw.yaml

```diff
@@ -0,0 +1,194 @@
+name: Gauss
+description: Three turbines using Gauss model
+floris_version: v3.0.0
+
+logging:
+  console:
+    enable: false
+    level: WARNING
+  file:
+    enable: false
+    level: WARNING
+
+solver:
+  type: turbine_grid
+  turbine_grid_points: 1
+
+farm:
```

Comment:
Remove farm layout since it's set internally from `ORBIT`. Same comment for the `05-offshore-h2`. Would be good to consolidate locations where we have redundant files in the code so changes don't get lost between files. We currently have 3 18MW turbines in `HOPP`.

Reply:
Irrelevant since the example has been removed.